src/theme.tsx

```diff
@@ -149,6 +179,21 @@ const TEXTS_DE: ThemeTexts = {
     counterQuantity: 'menge',
     counterMinValue: 'minimal',
     counterMaxValue: 'maximal',
+    timerDaysShortLabel: 'Tg.',
```

Comment:
Are all these translations in the spec? Have them been defined by Localization team?

Reply:
Yes